modules/message-lane/src/inbound_lane.rs

```diff
@@ -55,29 +55,32 @@ impl<S: InboundLaneStorage> InboundLane<S> {
 	/// Receive state of the corresponding outbound lane.
 	pub fn receive_state_update(&mut self, outbound_lane_data: OutboundLaneData) -> Option<MessageNonce> {
 		let mut data = self.storage.data();
-		if outbound_lane_data.latest_received_nonce > data.latest_received_nonce {
+		let last_received_nonce = data.last_delivered_nonce();
+
+		if outbound_lane_data.latest_received_nonce > last_received_nonce {
```

Comment:
Shouldn't this be `last_received_noce` for consistency reasons?

Reply:
I've changed the variable name to `last_delivered_nonce` for consistency, is that what you meant? I don't want to change the `OutboundLane` names, because of the reasons I gave in the description.

Reply:
Yeah. As a non-native speaker I'm always a bit confused about 'latest' and 'last'. So, personally instead of 'latest', I would prefer something like 'newest' or 'current'. But that is admittedly a personal preference.